demos/lineal_spring.py

```diff
@@ -42,20 +46,38 @@
 # are 'flattened' versions of the displacement and forcing vectors. By solving this
 # equation, we see how the structure of beams responds to the forcing.
 #
+# The main difference to note with the Laplacian smoothing approach is the following.
+# The Laplacian smoothing method is formulated in terms of a mesh velocity
```

Comment:
Just wondering maybe it helps to add some reference s to the Laplacian smoothing method as the lineal spring method, or some intuition about why the "mesh velocity" is govern by a poisson equation with RHS forces. As it seems that it is more intuitive to me that the second derivative of the displacement is force instead of the velocity.

Reply:
Thanks so much for this comment! It made me realise some fundamental flaws with the approach as-implemented. Will report back.

Reply:
What do you think of the motivation in 67d3955a3c3c7d05dff0ca22fefb9bc4794e55f4?

Reply:
Once this is merged, I will open an issue for introducing even simpler 1D demos for the different methods. We can move this motivation to the one for Laplacian smoothing.

Reply:
Thanks Joe! I think the motivation in https://github.com/mesh-adaptation/movement/commit/67d3955a3c3c7d05dff0ca22fefb9bc4794e55f4 is clear to me.